dashboard/modules/reporter/tests/test_reporter.py

```diff
@@ -113,13 +113,21 @@ def test_case_stats_exist():
             "ray_node_mem_total" in metric_names,
             "ray_raylet_cpu" in metric_names, "ray_raylet_mem" in metric_names,
             "ray_node_disk_usage" in metric_names,
+            "ray_node_disk_free" in metric_names,
             "ray_node_disk_utilization_percentage" in metric_names,
             "ray_node_network_sent" in metric_names,
             "ray_node_network_received" in metric_names,
             "ray_node_network_send_speed" in metric_names,
-            "ray_node_network_receive_speed" in metric_names
+            "ray_node_network_receive_speed" in metric_names,
+            "ray_cluster_active_nodes" in metric_names
         ])
 
+    # TODO: finish this unit test
+    def test_case_cluster_stats_exist():
+        autoscaler_report = 
+        ReporterAgent._record_stats(_, autoscaler_report)
```

Comment:
if you are going to make `_record_stats` non static method,  this part won't work

Reply:
discussed offline